examples/complete/fixtures.us-east-2.tfvars

```
@@ -6,4 +6,4 @@ stage = "test"
 
 name = "cloudfront-s3-cdn"
 
-parent_zone_name = "testing.cloudposse.co"
\ No newline at end of file
+parent_zone_name = "testing.cloudposse.com"
```

Testing domain is `.co` _[sic]_
```suggestion
parent_zone_name = "testing.cloudposse.co"

```